assets/js/views/ChargingSessions.vue

```diff
@@ -12,108 +12,107 @@
 
 		<div class="row">
 			<main class="col-12">
-				<div class="mb-4">
+				<div class="d-flex justify-content-start mb-4">
 					<a
 						class="btn btn-outline-secondary text-nowrap my-2"
-						:href="`./api/sessions?format=csv&amp;lang=${$i18n.locale}`"
+						:href="csvHrefLink()"
 						download="sessions.csv"
 					>
 						{{ $t("sessions.downloadCsv") }}
 					</a>
 				</div>
 
 				<div v-for="group in sessionsByMonthAndLoadpoint" :key="group.month">
-					<div class="mx-2">
-						<div class="d-flex align-items-baseline my-5">
-							<h2 class="me-4 mb-0">
-								{{ formatGroupHeadline(group.month) }}
-							</h2>
-						</div>
+					<div class="d-flex align-items-baseline my-5">
```

Comment:
```suggestion
					<div class="d-flex align-items-center my-5">
```

Dann ist der Kasten oben/unten Bündig mit der Überschrift.
![Bildschirm­foto 2023-01-30 um 15 34 24](https://user-images.githubusercontent.com/152287/215506588-191fe2f7-530d-4558-befb-c631aa702ba1.png)